portfolio/src/main/webapp/index.html

```diff
@@ -17,7 +17,27 @@ <h1> Alisha Kewalramani </h1>
            </ul>
       </nav>
       <div class="alt1">
-        <img src="images/alisha.jpeg"/>
+        <div class="slideshowContainer">
+            <div class="individualSlide">
+                <img src="images/alisha.jpeg"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic.JPG"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic2.jpg"/>
+            </div>
+            <div class="individualSlide">
+                <img src="images/solopic3.JPG"/>
+            </div>
+         </div>
+        <div class="dotContainer">
```

Comment:
suggestion: you could add these dots programmatically based on the number of individualSlide elements in slideshowContainer, so that you don't need to remember to add a dot whenever a new slide is added

Reply:
Good idea I will add this!